mkosi.conf.d/fedora/10-fedora.conf

```diff
@@ -24,6 +24,7 @@ Packages=
         libasan
         libbpf
         libcap-ng
+        libcap-ng-utils
```

Comment:
I went through a lot of effort to get the packages in sync across all distros so can we please add these for the other distros as well. https://pkgs.org/ is quite nice for searching packages